resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/CircuitBreakerConfig.java

```diff
@@ -363,6 +373,28 @@ public Builder slowCallDurationThreshold(Duration slowCallDurationThreshold) {
             return this;
         }
 
+        /**
+         * Configures CircuitBreaker with a fixed wait duration which controls how long the
+         * CircuitBreaker should stay in Half Open state, before it switches to open. This is an
+         * optional parameter.
+         *
+         * By default CircuitBreaker will stay in Half Open state until
+         * {@code minimumNumberOfCalls} is completed with either success or failure.
+         *
+         * @param waitDurationInHalfOpenState the wait duration which specifies how long the
+         *                                CircuitBreaker should stay in Half Open
+         * @return the CircuitBreakerConfig.Builder
+         * @throws IllegalArgumentException if {@code waitDurationInOpenState.toMillis() < 1000}
+         */
+        public Builder waitDurationInHalfOpenState(Duration waitDurationInHalfOpenState) {
```

Comment:
I think we should rename it to `maxWaitDurationInHalfOpenState`? Because the wait duration is shorter if minimumNumberOfCalls have been processed.

Reply:
sure make sense.